components/locale/src/langid.rs

```diff
@@ -128,18 +134,18 @@ impl FromStr for LanguageIdentifier {
 
 impl std::fmt::Display for LanguageIdentifier {
     fn fmt(&self, f: &mut std::fmt::Formatter) -> std::fmt::Result {
-        self.language.fmt(f)?;
+        std::fmt::Display::fmt(&self.language, f)?;
```

Comment:
Why did you replace that?

Reply:
I was seeing errors like this:

```
error[E0034]: multiple applicable items in scope
   --> components/locale/src/langid.rs:135:23
    |
135 |         self.language.fmt(f)?;
    |                       ^^^ multiple `fmt` found
    |
note: candidate #1 is defined in an impl of the trait `std::fmt::Display` for the type `subtags::language::Language`
   --> components/locale/src/subtags/language.rs:140:5
    |
140 |     fn fmt(&self, f: &mut std::fmt::Formatter) -> std::fmt::Result {
    |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
note: candidate #2 is defined in an impl of the trait `std::fmt::Debug` for the type `subtags::language::Language`
   --> components/locale/src/subtags/language.rs:33:19
```

But that must have been a symptom of another problem because I reverted the changes and I'm not seeing those this morning.